vscode/src/webview/types.ts

```diff
@@ -0,0 +1,86 @@
+export interface Incident {
```

Comment:
It seems "our" version of the [`Diagnostic`](https://github.com/microsoft/vscode-languageserver-node/blob/8c5b095be0bb0fa28dac28f4d97512e489b38884/types/src/main.ts#L688).
Do we want to align them more?

Reply:
I'd like to move toward this - yes. Would like to iterate further on this but currently the Incident type covers what we are receiving from Kantra.